src/main/groovy/de/dkfz/roddy/tools/RoddyIOHelperMethods.groovy

```diff
@@ -362,4 +362,21 @@ class RoddyIOHelperMethods {
     static String printTimingInfo(String info, long t1, long t2) {
```

Comment:
Can you comment this method? It is completely unclear what info, t1 and t2 may possibly be.